### PR TITLE
Short function spacing sniff

### PR DESCRIPTION
**Pull request by KelvinJardin**

### CODE REVIEW

File: DatacodeStandard/Sniffs/Classes/OpeningBraceSniff.php [hunk 1]
```diff
@@ -11,7 +11,6 @@
  * Ensures no blank line after class open closure
  */
 class OpeningBraceSniff implements Sniff {
-
```
Comment (betterphp) [inline DatacodeStandard/Sniffs/Classes/OpeningBraceSniff.php]: No need to remove this line break really

File: DatacodeStandard/Sniffs/Commenting/FunctionCommentSniff.php [hunk 2]
```diff
@@ -14,7 +14,6 @@
  * Same as the Squiz standard, but skips validation for @inheritDoc comments
  */
 class FunctionCommentSniff extends SquizFunctionCommentSniff {
-
```
Comment (betterphp) [inline DatacodeStandard/Sniffs/Commenting/FunctionCommentSniff.php]: No need to remove this line break

File: DatacodeStandard/Sniffs/Functions/ShortFunctionSpacingSniff.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * Checks there is a space after a short function (fn)
+ *
+ * Same as the Squiz standard, but skips validation for @inheritDoc comments
+ */
+class ShortFunctionSpacingSniff implements Sniff {
+	public function register() {
```
Comment (betterphp) [inline DatacodeStandard/Sniffs/Functions/ShortFunctionSpacingSniff.php:15]: Should have a blank line above this function

File: DatacodeStandard/Sniffs/Functions/ShortFunctionSpacingSniff.php [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+namespace DatacodeStandard\Sniffs\Functions;
+
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+
+/**
+ * Checks there is a space after a short function (fn)
+ *
+ * Same as the Squiz standard, but skips validation for @inheritDoc comments
```
Comment (betterphp) [inline DatacodeStandard/Sniffs/Functions/ShortFunctionSpacingSniff.php]: This comment doesn't really make sense for this sniff - looks like it was copied from the doc comments one.

File: DatacodeStandard/Sniffs/PHP/RequireStrictTypesSniff.php [hunk 5]
```diff
@@ -13,13 +13,20 @@
  * This only checks for the statemenet existing and not that it has been used correctly
  */
 class RequireStrictTypesSniff implements Sniff {
-
```
Comment (betterphp) [inline DatacodeStandard/Sniffs/PHP/RequireStrictTypesSniff.php]: This blank line should be here

**Review by betterphp — CHANGES_REQUESTED** [review]

File: DatacodeStandard/Sniffs/Functions/ShortFunctionSpacingSniff.php (hunk 4 above)
Comment (KelvinJardin) [reply DatacodeStandard/Sniffs/Functions/ShortFunctionSpacingSniff.php]: nooob! XD

**Review by betterphp — APPROVED** [review]

**Pull request merged**